experimental/language-generation/README.md

```diff
@@ -95,6 +95,15 @@ Load the template manager with your .lg file(s)
 
 For C#
 
+```
+    TemplateEngine lgEngine = new TemplateEngine().AddFiles(pathToLGFiles); 
+```
+
+For NodeJS
+```
+    let lgEngine = new TemplateEngine().addFiles(pathToLGFiles);
+=======
+
 ```c#
     TemplateEngine lgEngine = TemplateEngine.FromFiles(pathToLGFile);
```

Comment:
These should be removed - right? FromFiles is no longer valid for both C# and JS